### PR TITLE
Artifact Object Number Pad UI

### PR DESCRIPTION
**Pull request by anjusr**
Artifact Object Number Pad UI

### CODE REVIEW

**Review by jageet — CHANGES_REQUESTED** [review]

File: QatarMuseum/ViewController/FloorMapViewController.swift [hunk 1]
```diff
@@ -243,7 +241,13 @@ class FloorMapViewController: UIViewController,GMSMapViewDelegate,ObjectPopUpPro
             showMarker(marker: l2Marker11, position: l2_atr11, titleString: "HelloWorld", imageName: "artifactimg", zoomValue: zoom)
             showMarker(marker: l2Marker12, position: l2_atr12, titleString: "HelloWorld", imageName: "artifactimg", zoomValue: zoom)
             showMarker(marker: l2Marker13, position: l2_atr13, titleString: "HelloWorld", imageName: "artifactimg", zoomValue: zoom)
-
+        } else if(level == levelNumber.three) {
```
[the remark this answers is not in the document]
Comment (anjusr) [reply QatarMuseum/ViewController/FloorMapViewController.swift]: its not mine.. this i edited to remove conflict

File: QatarMuseum/ar.lproj/Localizable.strings [hunk 2]
```diff
@@ -83,3 +83,8 @@
 "LANG_ENGLISH" = "en";
 "LANG_ARABIC" = "ar";
 "FLOOR_MAP_TITLE" = "MIA FLOORS";
+"ARTIFACT_NUMBERPAD_TITLE" = "OBJECT SEARCH";
+"OBJECT_TITLE" = "Enter The Object Number";
+"OBJECT_INFO" = "next to         to view the details";
```
[the remark this answers is not in the document]
Comment (anjusr) [reply QatarMuseum/ar.lproj/Localizable.strings]: to place tat image in btw

File: QatarMuseum/ViewController/FloorMapViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (jageet) [reply QatarMuseum/ViewController/FloorMapViewController.swift]: Remove this. This is redundant.

File: QatarMuseum/ar.lproj/Localizable.strings (hunk 2 above)
[the remark this answers is not in the document]
Comment (jageet) [reply QatarMuseum/ar.lproj/Localizable.strings]: Ok fine. Will this be constant in all the devices?

[the remark this answers is not in the document]
Comment (anjusr) [reply QatarMuseum/ar.lproj/Localizable.strings]: yup

File: QatarMuseum/ViewController/FloorMapViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (anjusr) [reply QatarMuseum/ViewController/FloorMapViewController.swift]: wat to remove here

**Pull request merged**